src/panels/lovelace/views/hui-view.ts

```diff
@@ -106,6 +118,16 @@ export class HUIView extends UpdatingElement {
 
     if (configChanged && !this._layoutElement) {
       this._layoutElement = createViewElement(viewConfig!);
+      this._layoutElement.addEventListener(
+        "view-create-card",
+        this._showCreateCardDialog
+      );
+      this._layoutElement.addEventListener("view-edit-card", (ev) =>
+        this._showEditCardDialog(ev)
```

Comment:
No need to call a function. You can just define it in here. 

We should dynamically import these.